tools/boilerplate/verify-boilerplate.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
The existing `golangci-lint` can do this too, https://golangci-lint.run/usage/linters/#goheader no need to introduce a bunch more Python and Bash.

Reply:
Done, converted. There are now two places to update the boilerplate though, `tools/linter/golangci-lint/.golangci.yml` (golangci-lint config) and `tools/boilerplate.txt` (controller-gen template). The former expects no comment marks, while the latter requires them.

Reply:
After doing this and rebasing, there were two files missing headers, but the golangci-lint header check doesn't tell you the names, just that there are files missing headers, when it runs in Github Actions, anway. I spent about an hour poking at it, but I've reverted to `boilerplate.py`, since that actually has useful error messages.

If we want to change it later, that's fine, but someone else will need to have a crack.